UltraStar Play/Packages/playshared/Runtime/UI/ItemPicker/EnumFlagsFieldRuntime.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Linq;
+using ProTrans;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+public static class EnumExtensions
+{
+    public static object SetFlag(this Enum enumValue, Enum flag)
```

Comment:
-  imprecise, could return `int`. But then below there are compiler errors. Can you throw in generics to fix it? If no, then leave as is.